src/App.js

```diff
@@ -215,7 +232,7 @@ class App extends Component {
           </SplitScreen.StaticPane>
           <SplitScreen.SlidingPane>
               <SortBar sortByDistance={this.sortByDistance} sortByAlphabet={this.sortByAlphabet} haveCoords={this.state.haveCoords}/>
-              <ResultList data={this.state.orgs} haveCoords={this.state.haveCoords} currentPos={this.state.position}/>
+              <ResultList ref ={instance => {this.resultListItem = instance}} cardClick={this.cardClick} data={this.state.orgs} haveCoords={this.state.haveCoords} currentPos={this.state.position}/>
```

Comment:
I think we should add a style that changes the cursor when it hovers over the list but this is just UI stuff. Looks good!

Reply:
I think that's a great idea! We can add that detail when we focus on UI in the coming weeks

Reply:
If you're talking about those cluster that are still there despite clicking on them, I think they're some kind of bug that has to do with the data.